res/Sectors/M1105/Gzirr!k'l.txt

```diff
@@ -0,0 +1,562 @@
+1624 Qrukewyr             E583988-6 Hi Pr                                 - - - 704   Kc M0 V M9 V       
+1625 Sposex               D000696-9 As Va Na Ni                           - - - 310   Na G5 V M2 V       
+1626 Bynav                E3269B9-6 Hi In                                 - - - 810   Na G8 V            
+1630 Kubroslax            D9A57B9-5 Fl                                    - - - 502   Na K4 V            
+1722 Oobregh!r            B5265QK-7 Ni                                    - - - 904   Kk M4 V            
+1723 Treeiraa             E400000-0 Ba Va                                 - K - 015   Kk G4 V            
+1725 Gnujoll              E000200-7 As Va Lo                              - - A 102   Na G1 V            
+1726 Stycet               C8A0543-9 He Ni                                 - - - 805   Na G9 II           
+1728 Kubroslax            C140220-9 De Lo Po                              - - - 804   Na K6 V            
+1729 Seflun               E957234-8 Lo                                    - - - 503   Na M9 V            
+1822 Xkarrghatii          E7A0000-0 Ba He                                 - O - 003   Kk M9 V M2 V       
+1823 Gyzimos              X510100-2 Lo                                    - - R 622   Na G7 V M8 V       
+1824 Rislackia            EAF7522-6 Ni                                    - - - 304   Na F4 V            
+1825 Bweweprockia         D866300-9 Ga Lo                                 - - - 700   Na K1 IV M4 V      
+1921 K'greellu            C99A1PK-5 Lo Wa                                 - - - 900   Kk K6 V            
+1922 Gnakgrai             D4252PK-7 Lo                                    - K - 900   Kk F9 V M8 V       
+1924 Starev               D8C3799-A Fl                                    - - - 514   Na M1 V            
+1927 Bjejox               A260675-B De Ni Ri                              - - - 904   Kr F0 V M5 V       
+1929 Kragrenk             E225000-0 Ba                                    - - R 010   Kr M0 V M8 V M5 V  
+2025 Thoswed              E9B5000-0 Ba Fl                                 - - R 022   Kr G1 V            
+2026 Ybrofos              A254740-D Ag                                    - N - 105   Kr M9 II K4 V      
+2029 Cruhados             C758545-8 Ag Ni                                 - - - 814   Kr M6 V M3 V       
+2121 Xkarrghatii          C898324-5 Lo                                    - - - 602   Kk G4 V            
+2222 Athabbus             E533000-0 Ba Po                                 - - - 004   Na K2 V M3 V       
+2223 Snognex              C78A677-D Ni Ri Wa                              - - - 804   Na M1 V M9 V M1 V  
+2224 Slovlagral           D646677-8 Ag Ni                                 - - - 702   Na F3 V            
+2225 Uzepwot              A788753-D Ag Ri                                 - N - 603   Kr F3 III          
+2226 Gicung               X9B9332-4 FL Lo                                 - - R 812   Kr K1 V            
```

Comment:
This trade code should be "Fl", not "FL"

Reply:
Fixed